### PR TITLE
fix a few typos in README, some tidying up

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PASSED

Started: 2017-11-01 00:58:27
Finished: 2017-11-01 01:01:31


<h2>Failing Jobs</h2>
<ul>

<li>safari:10.0</li>

<li>MicrosoftEdge:14.14393</li>

</ul>




View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/20067)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/295558470)

File: README.md [hunk 1]
```diff
@@ -48,22 +52,23 @@ excluded from your proxy lookups.
 Running Tests Manually
 ======================
 
-The test server can be started using
+The test server can be started using this command:
 
     ./wpt serve
```
Comment (foolip) [inline README.md]: Can you use the ``` style for this as well, and either use sh as the language everywhere, or nowhere?

File: README.md [hunk 2]
```diff
@@ -48,22 +52,23 @@ excluded from your proxy lookups.
...
+
+To override the default ports, copy the `config.default.json` file to
+`config.json`, and replace the portion in the new file that reads
 
 ```
 "http": [8000, "auto"]
 ```
 
-to some port of your choice e.g.
+to some port of your choice – e.g.,
```
Comment (foolip) [inline README.md]: Comma instead of – would seem more typical.

File: README.md [hunk 3]
```diff
@@ -75,46 +80,46 @@ Running Tests Automatically
 Tests can be run automatically in a browser using the `run` command of
 the `wpt` script in the root of the checkout. This requires the hosts
 file setup documented above, but you must *not* have the
-test server already running when calling `wpt run`. The basic command
+test server already running when calling `wpt run`. The basic command-
```
Comment (foolip) [inline README.md]: This turns into "command- line", need to put command-line together.

File: README.md [hunk 4]
```diff
@@ -75,46 +80,46 @@ Running Tests Automatically
...
-On other platforms, download the firefox archive and common.tests.zip
-archive for your platform from
-[https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/](Mozilla CI)
+On other platforms, download the Firefox archive and respective
+`*.common.tests.zip` archive for your platform from [Mozilla CI](https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/).
 
-Then extract `certutil[.exe]` from the tests.zip package and
-`libnss3[.so|.dll|.dynlib]` and put the former on your path and the latter on
-your library path.
+Then extract `certutil[.exe]` from the `*.common.tests.zip` package, and make
```
Comment (foolip) [inline README.md]: @jgraham, can you check if this is correct?

File: README.md [hunk 5]
```diff
@@ -364,10 +372,10 @@ because we have to go through all the tests and ensure that they match
 the specification correctly. But we look at all of them, and take
 everything that we can.
 
-OWNERS files are used only to indicate who should be notified of pull
+`OWNERS` files are used only to indicate who should be notified of pull
```
Comment (foolip) [inline README.md]: I think I'd leave these unquoted, like in docs/introduction.md.

**Review by foolip — COMMENTED** [review]
Thanks for submitting these fixes, and sorry that nobody's noticed it until now!

File: README.md (hunk 1 above)
Comment (cvan) [reply README.md]: yes, totally - I’d prefer that as well

File: README.md (hunk 2 above)
Comment (cvan) [reply README.md]: yes, good point!

File: README.md (hunk 3 above)
Comment (cvan) [reply README.md]: ah, good catch - thanks

File: README.md (hunk 5 above)
Comment (cvan) [reply README.md]: aye, understood

**Comment by Hexcles** [comment]
This PR has gone stale. Besides, REAME has been overhauled.

**Pull request closed without merging**